build-system/server/typescript-compile.js

```diff
@@ -13,40 +13,55 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-const pathModule = require('path');
+const esbuild = require('esbuild');
+const globby = require('globby');
+const path = require('path');
 const {cyan, green} = require('kleur/colors');
+const {endBuildStep} = require('../tasks/helpers');
 const {exec} = require('../common/exec');
 const {log} = require('../common/logging');
 
-const SERVER_TRANSFORM_PATH = 'build-system/server/new-server/transforms';
-
-/**
- * @return {string}
- */
-function getBuildCmd() {
-  switch (process.platform) {
-    case 'win32':
-      return `node .\\node_modules\\typescript\\lib\\tsc.js -p ${SERVER_TRANSFORM_PATH.split(
-        '/'
-      ).join(pathModule.sep)}${pathModule.sep}tsconfig.json`;
-
-    default:
-      return `./node_modules/typescript/bin/tsc -p ${SERVER_TRANSFORM_PATH}/tsconfig.json`;
-  }
-}
+const SERVER_TRANSFORM_PATH = path.join(
+  'build-system',
+  'server',
+  'new-server',
+  'transforms'
+);
+const CONFIG_PATH = path.join(SERVER_TRANSFORM_PATH, 'tsconfig.json');
```

Comment:
Curious: Does this break if you don't construct the path like this? If so, I think it's more readable to use `path.normalize()` where necessary and retain the original `'build-system/server/new-server/transforms'` here.

Reply:
It actually breaks because I constructed the path this way! As you informed in chat, `globby.sync` requires posix style paths.